pyfxa_versions/__init__.py

```diff
@@ -55,7 +55,7 @@ def get_info(site):
 @click.command()
 @click.option('-c',
               '--config',
-              default=f'{pkg_resources.resource_string(__name__, "pyfxa-versions.json")}',
+              default=f'{pkg_resources.resource_string(__name__, "data/pyfxa-versions.json")}',
```

Comment:
I fixed this to point to the right location for the .json file.

I have no idea if you plan to run this script on Windows. If you do, then this might be Windows-unfriendly and you'll need to fix it.

Reply:
thanks, I left this dangling in my rush to deliver :)